internal/binutils/binutils.go

```diff
@@ -140,7 +144,87 @@ func initTools(b *binrep, config string) {
 		b.addr2line, b.addr2lineFound = findExe("gaddr2line", append(paths["addr2line"], defaultPath...))
 	}
 	b.nm, b.nmFound = findExe("nm", append(paths["nm"], defaultPath...))
-	b.objdump, b.objdumpFound = findExe("objdump", append(paths["objdump"], defaultPath...))
+	// MacOS binary is named objdump, Linux binary is named llvm-objdump-version
+	b.objdump, b.llvmObjdumpFound, b.objdumpFound = findObjdump(append(paths["llvm-objdump"], append(paths["objdump"], defaultPath...)...))
+}
+
+// findObjdump finds all binaries named objdump, llvm-objdump*. If multiple LLVM objdump are found,
+// it picks objdump with the latest major version. If no LLVM objdump is found, returns the path to
+// any binary named objdump, if found.
+func findObjdump(paths []string) (string, bool, bool) {
+	allObjdump := findAllObjdump("llvm-objdump*", paths)
```

Comment:
findExe uses exec.LookPath and findAllObjdump does not, which changes the semantics of how the paths are used here. To avoid having to document that subtle difference, can we avoid having to do the globbing and instead look up the objdump tools by their exact known names? We can list as many of them as needed, but I'm thinking just "llvm-objdump", "objdump", plus "gobjdump" on OSX should be enough.

Reply:
I removed this code since we will require llvm-objdump to be available.